app/views/handlers/content.py

```diff
@@ -45,8 +45,6 @@ def _get_content_title(self, transformed_block):
         if content:
             return self._get_safe_page_title(content["title"])
 
-    def _is_first_block_in_individual_response(self, location, schema):
-        section = schema.json.get("individual_response", {}).get(
-            "individual_section_id"
-        )
-        return location.block_id == schema.get_first_block_id_for_section(section)
+    def _is_block_first_block_in_individual_response(self, block_id, schema):
+        section_id = schema.get_individual_response_individual_section_id()
+        return block_id == schema.get_first_block_id_for_section(section_id)
```

Comment:
I realised we should be checking first loction on the path not the schema. So it should be something like:
```py
    def _is_individual_response_first_location(self):
        individual_section_id = self._schema.get_individual_response_individual_section_id()
        return (
            self.current_location.section_id == individual_section_id
            and self._current_location.block_id == self._routing_path[0]
        )
```
Feel free to get suggestion on method naming.

Reply:
In my defence, I did use ` location.block_id == routing_path[0]` at the beginning of this PR. Refactored, thanks 👍.